tests/OmniSharp.Roslyn.CSharp.Tests/CompletionFacts.cs

```diff
@@ -175,7 +175,7 @@ public Class1()
             {
                 while (completions.IsIncomplete)
                 {
-                    completions = await FindCompletionsAsync(filename, input, host);
+                    completions = await FindCompletionsAsync(filename, input, host, forceExpandedCompletionIndexCreation: false);
```

Comment:
We could likely avoid this entire loop if we just set the option to true.

Reply:
I ended up doing that for `FindCompletionsWithImportedAsync()`. Will let this test remain our test that the index will eventually be created.